docs/java-rest/low-level/configuration.asciidoc

```diff
@@ -25,7 +25,7 @@ seconds).
 include-tagged::{doc-tests}/RestClientDocumentation.java[rest-client-config-timeouts]
 --------------------------------------------------
 
-timeouts also can set by RequestOptions per request.this will override RestClient customizeRequestConfig.
+Timeouts also can set per request with RequestOptions. This will override RestClient customizeRequestConfig.
```

Comment:
Not sure what this is doing in with the release notes, but it looks like it's missing a word.
```suggestion
Timeouts also can be set per request with RequestOptions, which overrides RestClient customizeRequestConfig.
```

Reply:
Ah, I meant to mention that - in tracking down what one of the PRs was for, I found a change to the documentation that needed some improvement (but didn't improve enough 😁).